src/main/java/com/surrealdb/connection/model/RpcRequest.java

```diff
@@ -0,0 +1,21 @@
+package com.surrealdb.connection.model;
+
+import lombok.Getter;
+
+import java.util.UUID;
+
+/**
+ * @author Khalid Alharisi
+ */
+@Getter
+public class RpcRequest {
+    private final String id;
+    private final String method;
+    private final Object[] params;
+
+    public RpcRequest(String method, Object... params) {
+        id = UUID.randomUUID().toString(); // TODO: change to a faster random ID
```

Comment:
This doesn't need to be cryptographically secure. The Node.js and Python libraries just use an incrementing number for each request. This is only necessary to detect which response is for which request.